docs/_docs/milk-planning.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Milk Planning
+---
+
+# Milk Planning
+
+Every Saturday Raffaella has to estimate the quintals of milk necessary to produce all products
+for the following week.
+She makes this estimate by taking into account the following factors:
+
+- the quintals of milk processed in the same period of the previous year
+- the quintals of milk needed by the products that have to be produced in the following week
+  (this is made reading from a recipe book the the quintals of milk needed to produce a quintal of a given product)
+- the current stock
```

Comment:
Remove this
```suggestion
```

Reply:
We re-introduced the current stock as info for the milk reorder.